teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(name, age, parent_permission, specialization)
+    super(name, age, parent_permission)
+    @specialization = specialization
```

Comment:
- [ ] Same thing for the Teacher class  which is well created , but  the optional (or default) value for `name` and for `parent_permission` parameters are not defined, Kindly add them 👍 

```ruby
class Teacher < Person
 .
 .
 .
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
       #...
  end
 .
 .
 .
end
```